ringtail/storagemanager.py

```diff
@@ -2608,10 +2608,11 @@ def _generate_result_filtering_query(self, filters_dict):
                             self._ligand_substructure_position_filter(temp_lig_filter)
                         )
                     join_stmnt = " " + lig_filters["ligand_operator"] + " "
```

Comment:
I think Parentheses are also needed somewhere after this, to make sure multiple conditions are evaluated by the user specified logical_operator within their group, but eventually the groups (name, substructure, etc.) are always combined by "AND". When we finish the joining of multiple statements within a specific filter group, we could put Parentheses just to be safe.